core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -1184,7 +1184,8 @@ class TimeSeriesShard(val ref: DatasetRef,
         val tsp = part.asInstanceOf[TimeSeriesPartition]
         brRowReader.schema = schema.ingestionSchema
         brRowReader.recordOffset = recordOff
-        tsp.ingest(ingestionTime, brRowReader, overflowBlockFactory, maxChunkTime)
+        tsp.ingest(ingestionTime, brRowReader, overflowBlockFactory,
+          storeConfig.createChunkAtFlushBoundary, Option(flushBoundaryMillis), maxChunkTime)
```

Comment:
Dont create a new `Option` object for each sample. Will result in too many objects. Create a val and reuse.

Reply:
update the code, please take a look.